### PR TITLE
Chinese (Traditional) Translations

### PR DESCRIPTION
**Pull request by 1011025m**
This PR adds translations for Chinese (Traditional).

Some vocabulary might not fit in different cultures. Anyone who is a native speaker should take a look and give suggestions. Thank you!

### CODE REVIEW

**Comment by 1011025m** [comment]
String for the bootstrapper style preview is still hardcoded

**Comment by pizzaboxer** [comment]
I'll keep this PR open until localization support is fully finished.

File: Bloxstrap/Resources/Strings.zh-TW.resx [hunk 1]
```diff
@@ -0,0 +1,1023 @@
...
+    <value>Bloxstrap未能被正確安裝. 請您確定安裝位置是否：{0}</value>
+  </data>
+  <data name="InstallChecker.SeeWhatsNew" xml:space="preserve">
+    <value>按此查看新版本的新增功能</value>
+  </data>
+  <data name="InstallChecker.Updated" xml:space="preserve">
+    <value>Bloxstrap已更新到v{0}</value>
+  </data>
+  <data name="InstallChecker.VersionDifferentThanInstalled" xml:space="preserve">
+    <value>你開啓的Bloxstrap版本與已安裝的版本不符合.
```
Comment (KiminaNatsuki) [inline Bloxstrap/Resources/Strings.zh-TW.resx:512]: "不相符"是不是更好一點？

File: Bloxstrap/Resources/Strings.zh-TW.resx [hunk 2]
```diff
@@ -0,0 +1,1023 @@
...
+    <value>所有檔案</value>
+  </data>
+  <data name="Menu.AlreadyRunning" xml:space="preserve">
+    <value>Bloxstrap正在運行中, 可能在追蹤Roblox動態. 請留意：由於Roblox在運行中, 並非所有設定會即時套用. 你可以關閉所有Roblox客戶端來套用所有設定.</value>
+  </data>
+  <data name="Menu.Appearance.Bootstrapper.Description" xml:space="preserve">
+    <value>讓啓動器的外觀煥然一新... 或者回到過去.</value>
+  </data>
+  <data name="Menu.Appearance.Bootstrapper.Title" xml:space="preserve">
+    <value>啓動器</value>
```
Comment (KiminaNatsuki) [inline Bloxstrap/Resources/Strings.zh-TW.resx:585]: also will be suggesting to replace every mentioning of 啓動器 to 啟動器 in this PR

**Comment by 1011025m** [comment]
@KiminaNatsuki Thanks for the suggestions. I am not Taiwanese so some translations are based on my knowledge in Mandarin (with some Taiwanese references), apologies for that.

**Comment by KiminaNatsuki** [comment]
> @KiminaNatsuki Thanks for the suggestions. I am not Taiwanese so some translations are based on my knowledge in Mandarin (with some Taiwanese references), apologies for that.

No worries, I just saw the "Anyone who is a native speaker should take a look and give suggestions" part and decided to give some suggestion.

**Comment by pizzaboxer** [comment]
This translation PR has been closed because translation support has not yet been finished, and the translated strings are very likely to change, and we don't want to make too much of a mess here. Please contribute translations for your language when support is officially announced.

**Pull request closed without merging**